### ISSUE TITLE
examples error on flippy

### ISSUE THREAD

**Issue by Conlaoch** [treeform/fidget#60]
I followed the getting started directions and cloned the repo.  entered into the fidget directory and ran nimble install

nimble install
  Verifying dependencies for fidget@0.3.1
  <snipped>
  Info: Dependency on flippy@>= 0.2.0 already satisfied
  Verifying dependencies for flippy@0.4.1

Im using the newest version of nim
 nim --version
Nim Compiler Version 1.3.1 [MacOSX: amd64]
Compiled at 2020-04-16
Copyright (c) 2006-2020 by Andreas Rumpf

git hash: 9295251e68ee86b512de51a2f650729ac6893104
active boot switches: -d:release

when I try and run any of the examples I receive the following errors...

cd examples/bars
nim c -r bars.nim

<snipped>
Hint: textures [Processing]
/Users/mymachine/.nimble/pkgs/fidget-0.3.1/fidget/opengl/context.nim(194, 53) Error: undeclared identifier: 'Flippy'

using choosenim I've tried 1.0.6 and 1.3.1.  same error

**Comment by treeform** [comment]
Could you have some thing else named `flippy` in your path? 
`Flippy` is like on of the first things flippy defines?

**Comment by Conlaoch** [comment]
It's possible, but I cant check till this evening.

Last night I did a new git pull and followed the directions on the github
page.

cd fidget
nimble installcd examples/minimal
nim c -r minimal


The example then worked.  The only other change I made was I updated Nim to
the devel branch.  So it is at what.. 1.3.2 I think.

This is on OSx, and I had also done some homebrew cleanup and updating.

I can go to the github issue and update it with this info and then close it
if you want.
